px4

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import os
+import argparse
+
+# Dependencies:
+# Install MAVSDK: https://mavsdk.mavlink.io/develop/en/getting_started/installation.html
+
+# Usage:
+# Run SITL tests as in CI: ./px4 tests sitl
+
+# For convenience: You can add the directory to the PATH
+# Warning: If you work with multiple PX4 installations in
+# parallel, this can cause confusion as to what you are running
+# only recommended if a single directory is being used.
+
+# On MacOS, add this to your ~/.zshenv file:
+# export PATH=$PATH:~/src/PX4-Autopilot
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--verbose",
+                        help="Verbose output", default=False)
+    args = parser.parse_args()
+
+    # Change working directory to root working directory
+    os.chdir(os.path.dirname(os.path.abspath(__file__)))
+    os.system('DONT_RUN=1 make px4_sitl_default gazebo mavsdk_tests')
```

Comment:
@sfuhrer The two lines here show you how to build the MAVSDK tests and how to run them. Build dependencies are documented above.

Use AMC or QGC to watch the tests run in the background.